__tests__/app.test.js

```diff
@@ -177,6 +177,26 @@ describe.only("/api/articles", () => {
         expect(body.msg).toBe("Invalid Query");
       });
   });
+  test("GET:200 sends an array of articles by the topic value specified in the query", () => {
+    return request(app)
+      .get("/api/articles?topic=mitch")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.articles.length).toBe(12);
+        body.articles.forEach((article) => {
+          expect(article).toMatchObject({
+            author: expect.any(String),
+            title: expect.any(String),
+            article_id: expect.any(Number),
+            topic: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            article_img_url: expect.any(String),
+            comment_count: expect.any(Number),
+          });
+        });
+      });
```

Comment:
I think you should test for if there is not valid topic? I would you want your api to handle it?